Sources/OtlpGRPCSpanExporting/LogRecordExporter.swift

```diff
@@ -0,0 +1,66 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift OpenTelemetry open source project
+//
+// Copyright (c) 2023 Moritz Lang and the Swift OpenTelemetry project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import struct Foundation.Data
+import class Foundation.ProcessInfo
```

Comment:
What do you mean by "free"?

Reply:
I guess we already used foundation somewhere in this library? We try to avoid touching foundation if possible, but if you need `Date` then we're forced into it 🤔

Reply:
Ah, got it. Yeah, seems like I have to use `Data` here.